config.php

```diff
@@ -1,17 +1,11 @@
 <?php
-require_once "vendor/autoload.php";
 
-use Omnipay\Omnipay;
 
-define('CLIENT_ID', 'your client_id');
-define('CLIENT_SECRET', 'your client_secret');
+define('CLIENT_ID', 'AeQpypMHiyjhYOU98S1CLuwaLwfWv1J9H7leCqgu4RbqDaNk9lpiRWwnyX4wKZmRFIdm59oCzUXgn0mV');
```

Comment:
This should work. It is also the preferred method.

Change all occurrences of this.

```suggestion
const CLIENT_ID = 'AeQpypMHiyjhYOU98S1CLuwaLwfWv1J9H7leCqgu4RbqDaNk9lpiRWwnyX4wKZmRFIdm59oCzUXgn0mV';
```